test/Libraries/RevitNodesTests/Elements/ElementTests.cs

```diff
@@ -260,6 +260,32 @@ public void ElementCurveReferences_ExtractsExpectedReferences()
 
         #endregion
 
+        private static void GetHostedElements(Element elem)
+        {
+            var hostedElementsIncludeNothing = elem.GetHostedElements();
+            var hostedElementsIncludeOpenings = elem.GetHostedElements(true);
+            var hostedElementsIncludeEmbeddedWalls = elem.GetHostedElements(false, false, true, false);
+            var hostedElementsIncludeOpeningsAndEmbeddedWalls = elem.GetHostedElements(true, false, true, false);
```

Comment:
we need to test out all combinations, so missing a few where they're all `true`, first 3 `true`, etc